### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/v2-correction-birth/birth-make-correction-flow.spec.ts [hunk 1]
```diff
@@ -227,20 +227,28 @@ test.describe.serial('Birth Record correction flow', () => {
...
 
     await page.getByRole('button', { name: 'Confirm', exact: true }).click()
+
     await expectInUrl(page, `/events/overview/${eventId}`)
+
+    await page.waitForResponse((response) =>
+      response
+        .url()
+        .includes('/api/events/event.actions.correction.approve.request')
+    )
```
Comment (tareq89) [inline e2e/testcases/v2-correction-birth/birth-make-correction-flow.spec.ts:237]: learned something new!

**Review by tareq89 — APPROVED** [review]
